6.0.8 node.lib linking issues

<!--  As an open source project with a dedicated but small maintainer team, it can sometimes take a long time for issues to be addressed so please be patient and we will get back to you as soon as we can.
-->

### Preflight Checklist
<!-- Please ensure you've completed the following steps by replacing [ ] with [x]-->

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 6.0.8
* **Operating System:**
  * Windows Server 2016 and 2019
* **Last Known Working Electron version:**
  * 6.0.7

With the 6.0.8 binaries, native module builds are now failing with:

```
node.lib(node.exe) : fatal error LNK1223: invalid or corrupt file: file contains invalid .pdata contributions 
```

--- a/script/release/uploaders/upload-node-headers.py
+++ b/script/release/uploaders/upload-node-headers.py
@@ -56,7 +56,7 @@ def upload_node(bucket, access_key, secret_key, version):
       iojs_lib = os.path.join(DIST_DIR, 'win-x86', 'iojs.lib')
       v4_node_lib = os.path.join(DIST_DIR, 'win-x86', 'node.lib')
     elif get_target_arch() == 'arm64':
-      node_lib = os.path.join(DIST_DIR, 'node.lib')
+      node_lib = os.path.join(DIST_DIR, 'arm64', 'node.lib')
       iojs_lib = os.path.join(DIST_DIR, 'win-arm64', 'iojs.lib')
       v4_node_lib = os.path.join(DIST_DIR, 'win-arm64', 'node.lib')   
     else:

--- a/script/release/uploaders/upload-node-headers.py
+++ b/script/release/uploaders/upload-node-headers.py
@@ -55,6 +55,10 @@ def upload_node(bucket, access_key, secret_key, version):
       node_lib = os.path.join(DIST_DIR, 'node.lib')
       iojs_lib = os.path.join(DIST_DIR, 'win-x86', 'iojs.lib')
       v4_node_lib = os.path.join(DIST_DIR, 'win-x86', 'node.lib')
+    elif get_target_arch() == 'arm64':
+      node_lib = os.path.join(DIST_DIR, 'node.lib')
+      iojs_lib = os.path.join(DIST_DIR, 'win-arm64', 'iojs.lib')
+      v4_node_lib = os.path.join(DIST_DIR, 'win-arm64', 'node.lib')   
     else:
       node_lib = os.path.join(DIST_DIR, 'x64', 'node.lib')
       iojs_lib = os.path.join(DIST_DIR, 'win-x64', 'iojs.lib')

